src/blurbs/burstBlurb.js

```diff
@@ -0,0 +1,4 @@
+const burstBlurb =
```

Comment:
I wouldn't put these blurbs in separate files -- as the application grows, this becomes untenable.

Toss the blurbs directory and blurbs file and just put the text in the component => render() itself, set to an appropriately named variable (ie, burstInfo) that is passed into the InfoIcon component.

Reply:
Done